pyspecProcScripts/simple_functions.py

```diff
@@ -158,8 +159,13 @@ def find_apparent_anal_freq(s):
     """
     carrier = s.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
     dt = s["t"][1] - s["t"][0]
+    # In analytic signal the negative frequencies were tossed.
+    # So we first need to check if the carrier is in the SW of
+    # the analytic signal or if the signal was aliased
     if carrier < 1 / dt:
-        print("You are in the clear and no aliasing took place!")
+        psd.logger.info(
```

Comment:
this is no good --> logging should be imported independently -- it doesn't belong to pyspecdata